src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -323,6 +323,14 @@ public CSharpOperationFactory(SemanticModel semanticModel)
                         _ => null
                     };
                     return new NoneOperation(children, _semanticModel, boundNode.Syntax, type: type, constantValue, isImplicit: isImplicit);
+                case BoundKind.ValuePlaceholder:
```

Comment:
I don't think we want to generally handle placeholders; only placeholders we explicitly expect should be getting nodes.